app/views/campaigns/index.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Campaigns#index
+%p Find me in app/views/campaigns/index.html.haml
```

Comment:
If you are not using this, please remove it. Sam thing with the `index` action in the controller.